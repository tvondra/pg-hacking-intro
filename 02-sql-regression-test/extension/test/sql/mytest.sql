\set ECHO none

-- disable the notices for the create script (shell types etc.)
SET client_min_messages = 'WARNING';
\i sql/hacking--1.0.0.sql

\set ECHO all

SELECT hacking_function();

SELECT 1;
