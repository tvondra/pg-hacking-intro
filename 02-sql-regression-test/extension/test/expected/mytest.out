\set ECHO none
SELECT hacking_function();
 hacking_function 
------------------
 
(1 row)

SELECT 1;
 ?column? 
----------
        1
(1 row)

